.github/workflows/release.yml

```diff
@@ -18,6 +18,8 @@ jobs:
           node-version-file: '.nvmrc'
       - name: Install dependencies
         run: yarn install --frozen-lockfile
+      - name: Build
+        run: yarn build
```

Comment:
I think we are just missing telling `semantic-release` to check `style.css` and `script.js` into the repo. It should be enough to add them [here](https://github.com/zendesk/copenhagen_theme/blob/master/.releaserc#L13).

Reply:
Even easier! :)
Thanks for pointing it out as I'm not too familiar with `semantic-release`.